src/assets/icons/global/GlobalSvgSelector.tsx

```diff
@@ -0,0 +1,153 @@
+import React from "react";
+interface Props{
+  id: string;
+
+}
+export const GlobalSvgSelector = ({ id }: Props) => {
+  switch (id) {
+    case 'header-logo':
+      return (
```

Comment:
Take out svg to assets/img/<name-of-icon>
And import this as NameOfIcon 
```suggestion
      return <NameOfIcon/>
```